<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcomebuy</class>
 <widget class="QDialog" name="welcomebuy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;进货员&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>551</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>库存查询</string>
    </attribute>
    <widget class="QLineEdit" name="leSearchname">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSearchGoods">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>21</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>货物名称：</string>
     </property>
    </widget>
    <widget class="QTableView" name="tvGoods">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>491</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbBuyExit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>进货单查询</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>进货单编号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leSearchNum">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSearchInlist">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QTableView" name="tvInlist">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>481</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbModify">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbBuyExit_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbDelete">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>添加进货单</string>
    </attribute>
    <widget class="QPushButton" name="pbAdd">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbBuyExit_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>321</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>进货单编号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leInlistNum"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>货物编号：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leGoodsNum"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>数量：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leAmount"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbBuyExit</sender>
   <signal>clicked()</signal>
   <receiver>welcomebuy</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbBuyExit_2</sender>
   <signal>clicked()</signal>
   <receiver>welcomebuy</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbBuyExit_3</sender>
   <signal>clicked()</signal>
   <receiver>welcomebuy</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbDelete</sender>
   <signal>clicked()</signal>
   <receiver>welcomebuy</receiver>
   <slot>pbDelete_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbModify</sender>
   <signal>clicked()</signal>
   <receiver>welcomebuy</receiver>
   <slot>pbModify_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbAdd</sender>
   <signal>clicked()</signal>
   <receiver>welcomebuy</receiver>
   <slot>pbAdd_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSearchGoods</sender>
   <signal>clicked()</signal>
   <receiver>welcomebuy</receiver>
   <slot>pbSearchGoods_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSearchInlist</sender>
   <signal>clicked()</signal>
   <receiver>welcomebuy</receiver>
   <slot>pbSearchInlist_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pbSearchGoods_clicked()</slot>
  <slot>pbAdd_clicked()</slot>
  <slot>pbDelete_clicked()</slot>
  <slot>pbSearchInlist_clicked()</slot>
  <slot>pbModify_clicked()</slot>
 </slots>
</ui>
